tutorials/stats/2_pipeline_integration.py

```diff
@@ -0,0 +1,106 @@
+# %% [markdown]
+"""
+# 2. Pipeline Integration
+
+In the DFF ecosystem, extractor functions act as regular extra handlers (
+see the documentation for the `pipeline` module
```

Comment:
I think it's better to refer to pipeline tutorials (or to both).

Reply:
Done